solutions/usaco-1060.mdx

```diff
@@ -53,5 +53,83 @@ int main(){
 ```
 
 </CPPSection>
+	
+<JavaSection>
+	
+```java
+// Created by Sathvik Chundru with help from Justin Chen
+import java.util.*;
+import java.io.*;
+class Main {
+	public static void main(String[] args) {
+		Scanner sc = new Scanner(System.in);
+		int n = sc.nextInt();
+		int[] petals = new int[n];
+
+		for (int i = 0; i < n; i++) {
+
+			// storing each petal count in an array
+			// so we can keep track of which flower
+			// has how many petals based on the 
+			// index it is stored in
+
+			petals[i]=sc.nextInt();
+		}
+
+		int counter = 0;
+
+		// creating 2 nested arrays to brute force every 
+		// possible combination of flowers
+
+		for (int i = 0; i < n; i++) {
+			for (int j = i; j < n; j++) {
+
+				int flowerCount=j-i+1;
+				int sum=0;
+				for (int k = i; k <= j; k++) {
+					sum += petals[k];
+				}
+
+				// finding the sum of all petals up to 
+				// this point so we can find the average 
+
+				if (sum % flowerCount != 0) {
+					
+					// checking to see the average is integer;
+					// if not, we can save time and optimize
+					// by skipping this batch of flowers
+
+					continue;
+				}
+
+				// average = total sum of petals / total number 
+				// of flowers
+
+				int avg = sum / flowerCount;
+
+				for (int k = i; k <= j; k++) {
+					if (petals[k] == avg) {
+
+						// checking to see if any flowers 
+						// have the exact same number of 
+						// petals as the average; if they do, 
+						// then adding them to a counter
+
+						counter++;
+						break;
+					}
+				}
+			}
+		}
+						      
+		System.out.println(counter);
+		sc.close();
+	}
+
+}
+
+```
+				 
+</JavaSection>
```

Comment:
lines 68, 70, 75, 80, 83, 86, 95, 97, 101, 107, 109, 112, 117, 128, 130, and 132 are unnecessary whitespace

Reply:
fixed
thought it looked better but sans is an :orz: